components/registry/org.wso2.carbon.registry.activities.ui/src/main/resources/web/activities/activity-ajaxprocessor.jsp

```diff
@@ -41,6 +41,19 @@
         ActivityBean activityBean;
         String requestedPage = request.getParameter(UIConstants.REQUESTED_PAGE);
 
+        String sortOrder = request.getParameter("sortOrder");
+        String imgType = "";
+        String displayStr = "display:none;";
+
+        if (sortOrder!=null){
+            if (sortOrder.equals("DES")){
```

Comment:
Since initial loading of activity list page does not contain sort order as parameter, and the display string depends on the value of sort order. There for null check is required. Because null check is already done, then no need to replace with
"DES".equals(sortOrder) 

Thanks!